Analyzers/src/AnalyzerCore.C

```diff
@@ -759,6 +759,14 @@ void AnalyzerCore::initializeAnalyzerTools(){
   cfEst.SetDataYear(DataYear);
   cfEst.ReadHistograms();
 
+  TString current_pdf_name = "NNPDF31_nnlo_hessian_pdfas";
```

Comment:
To save memory, you need to call PDF maps with strings which are need in the analysis code at initializing step.

Reply:
It looks like one should modify AnalyzerCore to get different PDF set..? I think we'd better have a separate class for this to make it configurable and orgnaized. Let me try this.

Reply:
btw, where are the LHAPDF classes?